app/redux/modalSlice.tsx

```diff
@@ -12,6 +12,10 @@ export type EventDetailState = {
     | {
         type: "filter-panel";
       }
+    | {
+        type: "event-delete";
+        event: SerializableEvent;
+      }
```

Comment:
Is there a event-delete modal?

Reply:
Oh no. I forgot to delete that - I will delete it. That was written when Andres suggested a delete modal but you didn't like the idea